features/step_definitions/system_packages.rb

```diff
@@ -0,0 +1,11 @@
+Then(/^I should see this client as a registered system$/) do
+  step %[I should see a "#{@this_client_hostname}" text]
+end
+
+Then(/^I follow this client as a registered system$/) do
+  step %[I follow "#{@this_client_hostname}"]
+end
+
+Then(/^I press "(.*?)"$/) do |btn_title|
+  click_button(btn_title)
+end
```

Comment:
@isbm: All these step definitions already exist elsewhere so this part of the patch can effectively be dropped it seems (see [here] (https://github.com/SUSE/spacewalk-testsuite-base/blob/master/features/step_definitions/register_client_steps.rb#L53), [here] (https://github.com/SUSE/spacewalk-testsuite-base/blob/master/features/step_definitions/register_client_steps.rb#L64) and [here] (https://github.com/SUSE/spacewalk-testsuite-base/blob/master/features/step_definitions/navigation_steps.rb#L50)). Please make generally sure to not duplicate these with every new feature!